source/memory.tex

```diff
@@ -878,7 +878,8 @@
 \begin{itemdescr}
 \pnum
 \mandates
-\tcode{T} is an implicit-lifetime type\iref{basic.types.general}.
+\tcode{T} is an implicit-lifetime type\iref{basic.types.general}
+and not an incomplete type\iref{term.incomplete.type}.
```

Comment:
According to https://github.com/cplusplus/draft/pull/6114#discussion_r1105990427
> Interestingly "T is a complete type" doesn't seem to come with a cross reference elsewhere whereas "T is not an incomplete type" does seem to (to term.incomplete.type) which I'll add to this PR in the next revision.

but I see no such reference.
![1677008938](https://user-images.githubusercontent.com/21071787/220444083-4aad5669-875b-429b-9212-64c6d4151b03.png)

Reply:
I'm not sure if you're pointing out that the reference isn't in the paper, or isn't in the draft created after my pull request. I confirmed it's there after my pull request:

![image](https://user-images.githubusercontent.com/648276/220449149-1a0b6cfe-191c-4b9a-b06f-dbcc8bde1fef.png)

Reply:
Yes, I'm talking about the paper.

Reply:
I was adopting practice from elsewhere in the draft, especially elsewhere in `memory.tex`.

Reply:
That makes sense. Your phrasing of
> "T is not an incomplete type" does seem to

suggested to me that you didn't insert it yourself.

Reply:
Feel free to mark this as resolved.